ChineseFilter/sections/adservers.txt

```diff
@@ -7,6 +7,7 @@
 ! Bad: ||adssubdomain.legitwebsite.com^ (should be in adservers_firstparty.txt)
 !
 !
+||csjkb.com^
```

Comment:
Could you show this request via screenshot?

Reply:
I will delete because `||asserts.avple.tv/file/avple-asserts/ad.js` blocked a related content.